xml/cap_admin_minibroker.xml

```diff
@@ -22,8 +22,8 @@
   <link xlink:href="https://github.com/osbkit">Microsoft Azure team</link>. It
   provides a simple method to provision service brokers on Kubernetes clusters.
  </para>
- <important>
-  <title>Minibroker Upstream Services</title>
```

Comment:
These 2 lines are part of the updated Minibroker messaging (https://github.com/SUSE/doc-cap/pull/431) and should be left in.